lineage/Analyze.py

```diff
@@ -70,10 +70,12 @@ def run_Analyze_over(list_of_populations, num_states):
     list_of_populations: a list of populations that contain lineages
     num_states: an integer number of states to identify (a hyper-parameter of our model)
     """
-    output = []
-    for population_idx, population in enumerate(list_of_populations):
-        output.append(Analyze(population, num_states))
-    return output
+    input_args = [(population, num_states) for population in list_of_populations]
```

Comment:
Confused on if this is the right way to do. It's throwing an an error about how `Analyze` is expecting an additional argument of `numStates`, but I wrote in the previous line code to fix this, so I'm not sure...

Reply:
```
shakthi@aretha:~/Projects/CAPSTONE$ make
mkdir -p ./manuscript/figures
. venv/bin/activate && ./genFigures.py 3
concurrent.futures.process._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 239, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 198, in _process_chunk
    return [fn(*args) for args in chunk]
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 198, in <listcomp>
    return [fn(*args) for args in chunk]
TypeError: Analyze() missing 1 required positional argument: 'num_states'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./genFigures.py", line 14, in <module>
    ff = eval(nameOut + '.makeFigure()')
  File "<string>", line 1, in <module>
  File "/home/shakthi/Projects/CAPSTONE/lineage/figures/figure3.py", line 53, in makeFigure
    accuracies, tr = accuracy_increased_cells()
  File "/home/shakthi/Projects/CAPSTONE/lineage/figures/figure3.py", line 108, in accuracy_increased_cells
    results_holder = run_Results_over(output)
  File "/home/shakthi/Projects/CAPSTONE/lineage/Analyze.py", line 195, in run_Results_over
    res_holder = e.map(Results, output)
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 671, in map
    timeout=timeout)
  File "/usr/lib/python3.7/concurrent/futures/_base.py", line 587, in map
    fs = [self.submit(fn, *args) for args in zip(*iterables)]
  File "/usr/lib/python3.7/concurrent/futures/_base.py", line 587, in <listcomp>
    fs = [self.submit(fn, *args) for args in zip(*iterables)]
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 184, in _get_chunks
    chunk = tuple(itertools.islice(it, chunksize))
  File "/usr/lib/python3.7/concurrent/futures/process.py", line 483, in _chain_from_iterable_of_lists
    for element in iterable:
  File "/usr/lib/python3.7/concurrent/futures/_base.py", line 598, in result_iterator
    yield fs.pop().result()
  File "/usr/lib/python3.7/concurrent/futures/_base.py", line 428, in result
    return self.__get_result()
  File "/usr/lib/python3.7/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
TypeError: Analyze() missing 1 required positional argument: 'num_states'
make: *** [makefile:18: output/figure3.svg] Error 1
```